CMakeLists.txt

```diff
@@ -12,15 +12,12 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-cmake_minimum_required(VERSION 2.8.7)
-
-project(trower-base64)
+project(base64)
 
 add_definitions(-std=c99)
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Werror -Wall")
```

Comment:
What is your reasoning behind getting rid of all warnings being errors?  Are you running into problems with the RDK-B build?